src/OrleansPSUtils/Orleans.psd1

```diff
@@ -0,0 +1,24 @@
+﻿@{
+	RootModule = 'OrleansPSUtils.dll'
+	ModuleVersion = '1.2.3'
```

Comment:
is there a way to have this populated by the version of the OrleansPSUtils (effectively the same as the rest of Orleans)?